src/rulesets/Base/indexing.jl

```diff
@@ -1,3 +1,53 @@
+#####
+##### getindex(::Tuple)
+#####
+
+function frule((_, ẋ), ::typeof(getindex), x::Tuple, i::Integer)
+    return x[i], ẋ[i]
+end
+
+function frule((_, ẋ), ::typeof(getindex), x::Tuple, i)
+    y = x[i]
+    return y, Tangent{typeof(y)}(ẋ[i]...)
+end
```

Comment:
Do we need this rule?
I guess it avoids going through a bunch of indexing machinery